scripts/playbook.yml

```diff
@@ -29,11 +31,15 @@
         - cmake
         - python-psycopg2
         - libselinux-python
-        - git
         - bash-completion
         - vim
-    - name: install bower package manager
-      npm: name=bower global=yes
+    - name: install npm packages needed
+      npm: name={{ item }} global=yes
+      with_items:
+        - bower
+        - yarn
+        - gulp-cli
+        - webpack
```

Comment:
I suppose it is the change into 5.1, that uses yarn and webpack... I don't know about gulp, I just found it there.